Header header
int8 ball_in_view
float64 ball_center_x
int64 ball_width
float64 ball_distance
int8 goal_in_view
float64 goal_center_x
int64 goal_width
float64 goal_distance
int8 ball_goal_lined_up_maybe
int8 ball_goal_lined_up
